source/scpd/quorum/QuorumIntersectionChecker.d

```diff
@@ -19,6 +19,17 @@ import scpd.Cpp;
 import scpd.quorum.QuorumTracker;
 import scpd.types.Stellar_types;
 
+extern (C++, "_GLOBAL__N_1") {
```

Comment:
DDOC

Reply:
I'm sorry but I have no idea how I DDOC this. Could you let me know the right comment for that?

Reply:
```
// Quorum intersection checker implementation
// This method was originally implemented in C++ but since the changes to `NodeID` had to be moved to D.
```

Something like this.